Project_19/Project_19/Controllers/SetListViewController2.swift

```diff
@@ -20,12 +41,32 @@ class SetListViewController2: UITableViewController {
         navigationItem.largeTitleDisplayMode = .automatic
     }
     
+//    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+//        guard let ARViewVC = segue.destination as? ARViewController else { return }
+//        if let name = nameTextField.text, !name.isEmpty {
+//            ARViewVC.title = name
+//        }
+//    }
+    
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        if let destination = segue.destination as? ViewController,
-           let indexPath = tableView.indexPathForSelectedRow {
-                destination.selectedSet = setList[indexPath.row]
+        print("AAAAAAAAA")
+        if segue.identifier == "showExistingARView",
+           let ARViewVC = segue.destination as? ARViewController, let indexPath = tableView.indexPathForSelectedRow {
+            print("BBBBBBB")
+            print("Selected Set: ", indexPath.row)
+//            ARViewVC.selectedSet = setList[indexPath.row]
         }
+        print("CCCCCCCC")
```

Comment:
Could use more descriptive debug print statements. Did this testing end up working?